### ISSUE TITLE
does it follow Sciter update ?

### ISSUE THREAD

**Issue by ffred60** [sciter-sdk/go-sciter#192]
Hi,
Still learning Golang from time to time and looked at go-sciter several times.
now I could try to use it "really" (just personal exercises) and I have a question about go-sciter updating.

nearly nothing changed on the readme page since a long time. it always speaks about a "v4.0.0.0 and higher" support. it's ok, but so we can't know exactly how it match with Sciter SDK.
and if we look in the "include" directory, we can see "4.0.0.3" or "4.1.5", but the latest Sciter SDK is 4.2.6.10 (from 19/02/19. v4.1.5 is from 31/03/18..).
so, is it a problem to use a recent Sciter SDK with go-sciter and if not do we lack functions with go-sciter compare to Sciter SDK v4.2.6.10 ?
the pictures on the readme page are nice, but they come from Sciter website and have no relation with the samples of go-sciter.
I saw in the "issue forum" infos about the possible use of "packfolder", so should we use that instead of "rice", is there an example somewhere ?

sorry if my english is not clear, I do not want to give the impression of reproaching anything. just curious about what to expect with go-sciter.

thanks..

**Comment by pravic** [comment]
Well, Sciter SDK contains headers and binaries (dynamic libraries). And the headers are rarely changed.

So, if you take the latest Sciter dll/so/dylib, it will work. We don't lack any functions.


> we can't know exactly how it match with Sciter SDK.

We can: by comparing headers from the include folder with the official ones (https://github.com/c-smile/sciter-sdk/tree/master/include).

 As for the packfolder vs rice vs zip - it's totally up to you. The restest2 example is exactly about packfolder.

**Comment by ffred60** [comment]
Hi,
ok, great, thanks !!

I see that Sciter's include files are also of different versions and some time back for the most part, but the version numbers don't seems to have the same meaning as those of go-sciter ("sciter-x-script.h" for example, 3.3.1.0 for Sciter and 4.0.0.3 for go-sciter, but the files seems to be the same). it's probably more a marker of time for go-sciter, with the Sciter SDK number when the file was updated..
but it's ok, I had the wrong feeling of outdated files when I first saw the go-sciter header files with old version numbers because I had in mind that I should find all of them with the version number of the last SDK. now I understand I was wrong, sorry for that..

sorry about the restest2 example, did not see it before.. really nice and simple.  :-)
thanks.
